src/NHSD.BuyingCatalogue.Identity.UserDatabase/Deployment/CreateTestUsers.sql

```diff
@@ -1,38 +1,42 @@
-﻿IF NOT EXISTS(
+﻿DECLARE @aliceEmail AS nvarchar(50) = N'AliceSmith@email.com';
+DECLARE @bobEmail AS nvarchar(50) = N'BobSmith@email.com';
+
+IF NOT EXISTS(
   SELECT *
   FROM dbo.AspNetUsers
-  WHERE UserName IN ('alice', 'bob'))
+  WHERE UserName IN (@aliceEmail, @bobEmail))
 BEGIN
 	DECLARE @address AS nchar(108) = N'{ "street_address": "One Hacker Way", "locality": "Heidelberg", "postal_code": 69118, "country": "Germany" }';
 
 	DECLARE @aliceId AS nchar(36) = CAST(NEWID() AS nchar(36));
 	DECLARE @bobId AS nchar(36) = CAST(NEWID() AS nchar(36));
 
-	-- 'Pass123$'
+	DECLARE @aliceNormalizedEmail AS nvarchar(50) = N'ALICESMITH@EMAIL.COM';
```

Comment:
To save duplication here, I think you can UPPER case the email address variables. e.g. `DECLARE @aliceNormalizedEmail AS nvarchar(50) = UPPER(@aliceEmail);`